src/azure-cli/azure/cli/command_modules/storage/operations/account.py

```diff
@@ -481,12 +481,16 @@ def update_management_policies(cmd, client, resource_group_name, account_name, p
 
 
 # TODO: support updating other properties besides 'enable_change_feed,delete_retention_policy'
-def update_blob_service_properties(cmd, instance, enable_change_feed=None, enable_delete_retention=None,
-                                   delete_retention_days=None, enable_restore_policy=None, restore_days=None,
+def update_blob_service_properties(cmd, instance, enable_change_feed=None, change_feed_retention_days=None,
+                                   enable_delete_retention=None, delete_retention_days=None,
+                                   enable_restore_policy=None, restore_days=None,
                                    enable_versioning=None, enable_container_delete_retention=None,
                                    container_delete_retention_days=None, default_service_version=None):
     if enable_change_feed is not None:
-        instance.change_feed = cmd.get_models('ChangeFeed')(enabled=enable_change_feed)
+        if enable_change_feed is False:
+            change_feed_retention_days = None
+        instance.change_feed = cmd.get_models('ChangeFeed')(
+            enabled=enable_change_feed, retention_in_days=change_feed_retention_days)
```

Comment:
consider https://github.com/Azure/azure-cli/pull/16990/files#r582553648

Reply:
As I replied above, with `--enable-change-feed` without `--change-feed-days`, `enable_change_feed` will be `True` and `change_feed_retention_days` will be `None`.